docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -7,21 +7,19 @@ nav_order: 11
 
 # Azure Execution Provider (Preview)
 
-The Azure Execution Provider enables ONNX Runtime to invoke an remote Azure endpoint for inferenece, the endpoint must be deployed beforehand.
-To consume the endpoint, a model of same inputs and outputs must be loaded locally in the first place.
+The Azure Execution Provider enables ONNX Runtime to invoke a remote Azure endpoint for inference. The endpoint must be deployed beforehand.
+To consume the endpoint, a model of same inputs and outputs must be first loaded locally.
 
-One use case for Azure Execution Provider is small-big models. E.g. A smaller model deployed on edge device for faster inference,
-while a bigger model deployed on Azure for higher precision, with Azure Execution Provider, a switch between the two could be easily achieved.
-Again, the two models must have same inputs and outputs.
+One use case for Azure Execution Provider is for small-big models. E.g. A smaller model can be deployed on edge devices for faster inference,
+while a bigger model deployed on Azure for higher precision. Using the Azure Execution Provider, switching between the two can be easily achieved (assuming same inputs and outputs). 
```

Comment:
Do we need a line beak here?

Reply:
Where? There is already a line break above this line "One use case..."

Reply:
Sorry, I was unclear. Does the line break need to be removed here?